packages/core-components/src/lib/unsorted/viz/index.js

```diff
@@ -8,6 +8,7 @@ export { default as Bubble } from './Bubble.svelte';
 export { default as Chart } from './Chart.svelte';
 export { default as Column } from './Column.svelte';
 export { default as DataTable } from './DataTable.svelte';
+export { default as Dropdown } from './Dropdown.svelte';
```

Comment:
This component is an `atom`, we should not add new components to `unsorted` anymore